packages/studio-base/src/panels/ThreeDeeRender/renderables/PointClouds.ts

```diff
@@ -888,6 +904,7 @@ export class PointClouds extends SceneExtension<PointCloudHistoryRenderable> {
       this.renderables.set(topic, renderable);
     }
 
+    renderable.pushHistory(pointCloud, originalMessage, renderable.userData.settings, receiveTime);
```

Comment:
```suggestion
    if(renderable.userData.settings.decayTime > 0) {
      renderable.pushHistory(pointCloud, originalMessage, renderable.userData.settings, receiveTime);
    }  
```
Looks a little strange to see `pushHistory` if we don't know that decay is active.